src/lib/utils.ts

```
@@ -1,20 +1,14 @@
- * The query is percent-encoded so spaces and special characters are safe
- * in a href attribute.
- * @param query - The search term to look up.
- * @returns A fully-qualified `https://google.com/search?q=…` URL.
- */
-export function googleSearchLink(query: string) {
-  return `https://google.com/search?q=${encodeURIComponent(query)}`
+export function googleSearchLink(term: string) {
+  return `https://www.google.com/search?q=${encodeURIComponent(term)}`
+}
+
+export function googleSearchLink(term: string) {
```

Biome (2.4.11)

[error] Shouldn't redeclare 'googleSearchLink'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)